sqldb/sqlc/queries/invoices.sql

```diff
@@ -0,0 +1,100 @@
+-- name: InsertInvoice :one
+INSERT INTO invoices (                                                         
+    memo, amount_msat, cltv_delta, expiry, features,
+    payment_addr, preimage, hash, state, payment_request,                 
+    last_amount_paid_msat, is_amp, is_hodl, is_keysend, created_at
+) VALUES ( 
+    $1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14,
+    CURRENT_TIMESTAMP
+) RETURNING id;
+
+-- name: InsertInvoiceEvent :exec
+INSERT INTO invoice_events (
+        invoice_id, created_at, event_type
+) VALUES (
+    $1, CURRENT_TIMESTAMP, $2
+);
+
+-- name: GetInvoiceIDBySetID :one
+SELECT invoice_id
+FROM amp_payments
+WHERE set_id = $1;
```

Comment:
I'm not 100% sure, but this and the below queries might be improved by adding indexes on the columns. AFAIK `references()` does not create indexes on the columns.

Reply:
yes, indices were left of the first iteration (now they are already there, basically for whatever columns we use for filtering the whole table)